docs/mkdocs.yml

```diff
@@ -175,6 +175,7 @@ pages:
 - ['project/who-written-for.md', 'Project', 'README first']
 - ['project/set-up-prereqs.md', 'Project', 'Set up prerequisites'] 
 - ['project/set-up-dev-env.md', 'Project', 'Work with a development container'] 
+- ['project/windows-client-test-server.md', 'Project', 'Set up a Windows server for client testing'] 
```

Comment:
I _think_ using the word `server` in this context is misleading - `Windows system` will then also include normal windows versions too.